src/DataStructures/Variables.hpp

```diff
@@ -112,6 +137,15 @@ class Variables<tmpl::list<Tags...>> {
   const double* data() const noexcept { return variable_data_.data(); }
   //@}
 
+  /// \cond
```

Comment:
should this just be a regular comment and not a doxygen one?

Reply:
No, I want the Doxygen conditional to disable the code inside them from being registered. See [https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdcond]()